CHANGELOG.md

```diff
@@ -10,6 +10,8 @@ All notable changes to this project will be documented in this file. From versio
 
 ### Fixed
 
+- Fix incorrect ordering of overloaded functions when schemas differ by @joelonsql in #4554
+  + The `Ord` instance for `Routine` had a typo comparing `schema2 == schema2` instead of `schema1 == schema2`.
```

Comment:
What would be the observable effect for the user here? We should put that down in the changelog, not the code change.